backend/src/test/java/de/neuefische/teddyscoronadiaries/controller/WeatherControllerTest.java

```diff
@@ -110,42 +107,9 @@ public void getProvinceCapitalWeatherdataShouldThrowErrorWhenApiUnavailable() {
     }
 
 
-    private List<WeatherData> getWeatherDataList() {
-        return List.of(
-                WeatherData.builder()
-                        .id("123")
-                        .weatherStateName("Clear")
-                        .weatherStateAbbreviation("c")
-                        .applicableDate("2021-03-21")
-                        .minTemperature(5.9865)
-                        .maxTemperature(20.345)
-                        .currentTemperature(20.240000000000002)
-                        .windSpeed(8.930032549754765)
-                        .humidity(50)
-                        .build(),
-                WeatherData.builder()
-                        .id("345")
-                        .weatherStateName("Light Cloud")
-                        .weatherStateAbbreviation("c")
-                        .applicableDate("2021-03-22")
-                        .minTemperature(6.485)
-                        .maxTemperature(16.725)
-                        .currentTemperature(15.93)
-                        .windSpeed(2.884157589803547)
-                        .humidity(70)
-                        .build(),
-                WeatherData.builder()
-                        .id("567")
-                        .weatherStateName("Heavy Cloud")
-                        .weatherStateAbbreviation("hc")
-                        .applicableDate("2021-03-23")
-                        .minTemperature(3.875)
-                        .maxTemperature(13.86)
-                        .currentTemperature(12.065000000000001)
-                        .windSpeed(4.708858778127734)
-                        .humidity(71)
-                        .build()
-        );
+    private Weather getWeather() {
+        return new Weather(List.of(new WeatherContent("Rain", "Leichter Regen", "01d")),
```

Comment:
Builder ;) Don't force me to browse your models to understand what `"01d"` is supposed to be.

Reply:
fine, I'll use a builder just for you just so you know 🙈

Reply:
🙏